lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunication.cpp

```diff
@@ -1316,6 +1316,7 @@ std::string GDBRemoteCommunication::ExpandRLE(std::string packet) {
     } else {
       decoded.push_back(*c);
     }
+    c++;
```

Comment:
Having the `c++` here is equivalent to having it in the "increment" portion of the for loop construct, why move it here?